frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -600,6 +604,19 @@ function DashboardsList(): JSX.Element {
 				),
 				key: '1',
 			},
+			{
+				label: (
+					<a
+						className="create-dashboard-menu-item"
```

Comment:
does this work in the light mode correctly ? we do not need any changes for the same ?

Reply:
<img width="1728" alt="Screenshot 2024-08-16 at 11 52 59 AM" src="https://github.com/user-attachments/assets/fdd549a7-3d9d-4cf4-9be7-a91481c3fdbd">
<img width="1259" alt="Screenshot 2024-08-16 at 11 53 13 AM" src="https://github.com/user-attachments/assets/79549597-14e2-4e8b-82af-0753fa510920">
Sharing the light theme. @vikrantgupta25 did you see any issues from the code perspective or visually?